### DISCUSSION TITLE
Users willing to donate time to test Zavala

### DISCUSSION THREAD

**Discussion by latvia234** [vincode-io/Zavala#186] [General]
Once in a while I check the development of Zavala. Today updates are less frequently than in the beginning. Why? I don't know exactly but could [this](https://zavala.vincode.io/2022/02/10/Zavala_and_Micro.blog.html#dog-fooding-zavala) be one of the reasons?

[From the Vincode / Zavala blog:](https://zavala.vincode.io/2022/02/10/Zavala_and_Micro.blog.html#dog-fooding-zavala)
> One of the main problems with running a small open source project is getting it tested sufficiently. I use TestFlight to distribute test versions of Zavala so that users can test it out. My problem is that the pool of users willing to donate time to testing is too small.

One thing to keep in mind is that not everyone is a developer. Also not everyone knows about Testflight. Like me. How can users like me help you, @vincode-io? Does it help if I signup for Testflight if I do not have experience with it? Or do you prefer to only work with experienced testers?

**Comment by vincode-io** [comment]
Don’t worry. Zavala is still being actively maintained. Development just slowed down because I was traveling most of the first half of this year.

If you or anyone would sign up for TestFlight it would help me immensely. Here is the link to use:

https://testflight.apple.com/join/iFbND23m

I just put out a new version of Zavala. It is 2.0.8 and is waiting for Apple to review it. If you sign up now, you might not see it, but when the 2.0.8 version is ready for testing you will be notified through the TestFlight app.
